packages/node/README.md

```diff
@@ -30,41 +30,44 @@ app.use(readme.metrics('<<apiKey>>', (req, res) => ({
 
 Note that in order to generate this URL, an API request is made to ReadMe once a day, and cached to a local file in `node_modules/.cache/readmeio`, to retrieve your projects `baseUrl`. If this request to ReadMe fails, the `x-documentation-url` header will not be added to responses.
 
 If you wish to not rely on this cache, you can opt to supply a `baseLogUrl` option into the middleware, which should evaluate to the public-facing URL of your ReadMe project.
 
 ### Configuration Options
+
 There are a few options you can pass in to change how the logs are sent to ReadMe. These are passed in an object as the third parameter to the `readme.metrics`.
 
 ```js
 const readme = require('readmeio');
 const env = process.env.NODE_ENV;
 
-app.use(readme.metrics('', req => ({
-  id: req.<userId>,
+app.use(readme.metrics('<<apiKey>>', req => ({
+  apiKey: req.<userApiKey>,
   label: req.<userNameToShowInDashboard>,
   email: req.<userEmailAddress>,
 }), {
   development: env !== 'production',
   bufferLength: 1,
-  blacklist: [[arrayOfSensitiveKeysToOmit]],
-  whitelist: [[arrayofKeysOnlyToSend]],
+  denylist: [[arrayOfSensitiveKeysToOmit]],
+  allowlist: [[arrayofKeysOnlyToSend]],
 }));
 ```
 
-| Option | Use |
-| :--- | :--- |
-| development | **default: false** If true, the log will be separate from normal production logs. This is great for separating staging or test data from data coming from customers. |
-| bufferLength | **default: 10** By default, we only send logs to ReadMe after 10 requests are made. Depending on the usage of your API it make make sense to send logs more or less frequently. |
-| blacklist | **optional** An array of keys from your API requests and responses headers and bodies that you wish to blacklist from sending to ReadMe.<br /><br />If you configure a blacklist, it will override any whitelist configuration. |
-| whitelist | **optional** An array of keys from your API requests and responses headers and bodies that you only wish to send to ReadMe. |
-| baseLogUrl | **optional** This is the base URL for your ReadMe project. Normally this would be `https://projectName.readme.io` or `https://docs.yourdomain.com`, however if this value is not supplied, a request to the ReadMe API will be made once a day to retrieve it. This data is cached into `node_modules/.cache/readmeio`. |
+| Option       | Use                                                                                                                                                                                                                                                                                                                     |
+| :----------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I've never really been a fan of padding Markdown tables out like this because it's one more thing to manage when changing content in a row, but I'm not going to put up a stink about it.